packages/app-config/scripts/package-subtree-local.sh

```
@@ -0,0 +1,1201 @@
+        done < <(git diff --name-only --diff-filter=U 2>/dev/null)
+
+        if [ ${#git_conflict_files[@]} -gt 0 ]; then
+          has_git_conflicts=true
+        fi
+
+        if echo "$apply_error" | grep -q "patch does not apply"; then
+          has_rejected_patches=true
+          while IFS= read -r line; do
+            if [[ "$line" =~ error:\ (.+):\ patch\ does\ not\ apply ]]; then
+              local failed_file="${BASH_REMATCH[1]}"
+              failed_file="${failed_file#$WORKSPACE_LOCATION/$TARGET_RELATIVE/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)